app/dedicated-cloud-gateways/aws-private-link.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "AWS PrivateLink Peering"
+content_type: reference
+layout: reference
+description: |
+  Create a private connection between your AWS environment and {{site.konnect_short_name}} using AWS PrivateLink.
+products:
+  - gateway
+works_on:
+  - konnect
+api_specs:
+  - konnect/control-planes-config
+
+related_resources:
+  - text: Dedicated Cloud Gateways
+    url: /dedicated-cloud-gateways/
+  - text: AWS PrivateLink Interface Endpoints
+    url: https://docs.aws.amazon.com/vpc/latest/privatelink/create-interface-endpoint.html
+  - text: Amazon VPC Documentation
+    url: https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html
+---
+
+You can establish a private connection between {{site.konnect_short_name}} and your AWS environment using AWS PrivateLink. This provides secure communication between your Data Plane and the Control Plane, reducing data transfer costs and ensuring compliance.
```

Comment:
```suggestion
You can establish a private connection between {{site.konnect_short_name}} and your AWS environment using AWS PrivateLink. This provides secure communication between your Data Plane and the Control Plane, reducing data transfer costs and ensuring compliance.

You can configure this instead of AWS Transit Gateways to secure your connection.
```
I don't know if that last part is accurate, but from reading both, it seems like you configure one or the other but not both. And I wanted that to be explicit.